tests/cases/fourslash/tsxCompletionOnOpeningTagWithoutJSX1.ts

```diff
@@ -3,4 +3,4 @@
 //@Filename: file.tsx
 //// var x = </**/;
 
-verify.completions({ marker: "", exact: [completion.globalThisEntry, ...completion.globalsVars, "x", completion.undefinedVarEntry] });
+verify.completions({ marker: "", exact: [completion.globalThisEntry, ...completion.globalsVars, completion.undefinedVarEntry] });

```

Comment:
This was also weird

Reply:
This is the same as the previous one, it also existed before. and after this modification, there should be no 'x'.  Maybe I can optimize this test case